lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
+
+  if (err.code === 'SESSION_TIMEOUT') {
+    content.title = translate('errors.session.title');
```

Comment:
However much we refrain from joining the framework club we are still using HOF as a tool to build a large number of forms, which IMO should be standardised to a point.
But, I am not fussy, perhaps it is too much to ask that a user always provides a translations function.